### PR TITLE
chore: add make uninstall-e2e-telemetry

### PR DESCRIPTION
**Pull request by zirain**
this will help people clean up after e2e

### CODE REVIEW

File: tools/make/kube.mk [hunk 1]
```diff
@@ -128,6 +128,12 @@ prepare-e2e: prepare-helm-repo install-fluent-bit install-loki install-tempo ins
 	kubectl rollout status statefulset tempo -n monitoring --timeout 5m
 	kubectl rollout status deployment otel-collector -n monitoring --timeout 5m
 
+.PHONY: uninstall-e2e
+uninstall-e2e:
```
Comment (arkodg) [inline tools/make/kube.mk:132]: should it be `prepare-e2e-telemetry` and `uninstall-e2e-telemetry` ?

Comment (zirain) [reply tools/make/kube.mk]: done

**Review by arkodg — APPROVED** [review]
LGTM thanks !

**Pull request merged**